portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,51 +40,67 @@
   public class Comment {
     private String time, user, content;
 
-    public Comment(String user, String content) {
-      Date date = new Date();
-      SimpleDateFormat formatter = new SimpleDateFormat("MMM d, HH:MM a");
+    public Comment(String user, String content, String time) {
```

Comment:
You might consider putting this in a separate file if the Comment class ends up being a little more complex (e.g. if you start adding methods to it).